config/config.go

```diff
@@ -70,3 +78,18 @@ func Get() (*Config, error) {
 
 	return cfg, envconfig.Process("", cfg)
 }
+
+// GetDBEndpoint get sql endpoint
+func (c Config) GetDBEndpoint () string {
+	return fmt.Sprintf("%s:%s", c.RDSDBHost, c.RDSDBPort)
+}
+
+// GetLocalDBConnectionString returns local connection string
+func (c Config) GetLocalDBConnectionString () string {
+	return fmt.Sprintf("postgres://%s:%s@localhost:%s/%s", c.DPPostgresUserName, c.DPPostgresUserPassword, c.DPPostgresLocalPort, c.DPPostgresLocalDB)
+}
+
+// GetRemoteDBConnectionString returns remote connection string
+func (c Config) GetRemoteDBConnectionString (authToken string) string {
+	return fmt.Sprintf("host=%s port=%s user=%s password=%s dbname=%s", c.RDSDBHost, c.RDSDBPort, c.RDSDBUser, authToken, c.RDSDBName)
```

Comment:
is it possible to use similar connection string for both local and remote DB connection?

Reply:
Unfortunately not - the PGX Pool connections to RDS are using IAM authentication, using an auth token generated by `github.com/aws/aws-sdk-go/service/rds/rdsutils`. The local connection is using a plain old connection string with the db name and password.